org.eclipse.winery.frontends/pom.xml

```diff
@@ -118,7 +118,7 @@
                             <goal>npm</goal>
                         </goals>
                         <configuration>
-                            <arguments>ci</arguments>
+                            <arguments>install</arguments>
```

Comment:
Why did use install now? I guess we should stick to `ci` (as it is faster and uses the defined versions)[https://www.geeksforgeeks.org/difference-between-npm-i-and-npm-ci-in-node-js/]